tests/conftest.py

```diff
@@ -80,3 +83,70 @@ def sqlite_engine():
 @pytest.fixture
 def Calendar():
     return calendar.Calendar(0)
+
+
+@pytest.fixture
+def no_event_user(session):
+    """a user made for testing who doesn't own any event."""
+    user = create_user(
+        session=session,
+        username='new_test_username',
+        password='new_test_password',
+        email='new2_test.email@gmail.com',
+        language_id='english'
+    )
+
+    return user
```

Comment:
Don't create a new fixture, use one of the others.

Reply:
i have to use several different users since i'm testing the use of the mailing list.